### PR TITLE
man: drop comment about /etc/fstab being preferrable over .mount units

### PR DESCRIPTION
**Pull request by poettering**
Both are fine, let's not claim otherwise.

Fixes: #4961

### CODE REVIEW

File: man/systemd.mount.xml [hunk 1]
```diff
@@ -148,16 +148,12 @@
...
     <title><filename>fstab</filename></title>
 
-    <para>Mount units may either be configured via unit files, or via
-    <filename>/etc/fstab</filename> (see
-    <citerefentry project='man-pages'><refentrytitle>fstab</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-    for details). Mounts listed in <filename>/etc/fstab</filename>
-    will be converted into native units dynamically at boot and when
-    the configuration of the system manager is reloaded. In general,
-    configuring mount points through <filename>/etc/fstab</filename>
-    is the preferred approach. See
```
Comment (evverx) [inline man/systemd.mount.xml:158]: I'm reading https://github.com/systemd/systemd/pull/3152#issuecomment-216072070
And I think we shouldn't drop comment about /etc/fstab being preferrable over .mount units

Comment (poettering) [reply man/systemd.mount.xml:158]: Can you elaborate?

Comment (evverx) [reply man/systemd.mount.xml:158]: oh, sorry. sure.
Well, `/etc/fstab` just works. `*.mount`-units require some [hacks](https://github.com/systemd/systemd/pull/3152#issuecomment-216025727).
I guess we should document all these hacks first. And drop
> In general, configuring mount points through /etc/fstab is the preferred approach

second

**Comment by keszybz** [comment]
I don't think we should make this change. While you are of course perfectly correctly that both are equivalent, that note was added to discourage newbies from trying to convert their fstab to mount units to be "cool". It'd keep the advice as is, and people who know what they are doing will know to ignore it when necessary.

**Comment by poettering** [comment]
hmm, ok, but then let's close both this one and #4961?

**Comment by keszybz** [comment]
Yeah, let's do that.

**Pull request closed without merging**